source/Willow/Willow.class.st

```diff
@@ -0,0 +1,63 @@
+"
+I'm a shared pool providing access to constants and classes defined by the lilbrary
+"
+Class {
```

Comment:
Changing this shared pool definition to 2 different pools (WillowClassification and WillowConstant) might improve readability of the situations mentioned in other comments.